rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -1,87 +1,41 @@
 :_content-type: REFERENCE
 [id="rosa-release-notes"]
-= {product-title} 4.10 release notes
+= What's new with {product-title}
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-release-notes
 
 toc::[]
 
 {product-title} (ROSA) is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
 
-[id="about-this-release-section_{context}"]
-== About this release
+{product-title} clusters are available on the link:https://console.redhat.com/openshift[Hybrid Cloud Console]. With the Red Hat OpenShift Cluster Manager application for ROSA, you can deploy {product-title} clusters to either on-premises or cloud environments.
 
-{product-title} https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.24 with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} 4.10 are included in this topic.
+[id="updating_{context}"]
+== Updating the ROSA CLI tool
 
-{product-title} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy {product-title} clusters to either on-premise or cloud environments.
+You can download the latest version of the ROSA CLI from the Hybrid Cloud Console.
 
-[id="about-this-release_{context}"]
-== ROSA layered and dependent component support and compatibility
-The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+To update to the latest version of the ROSA CLI:
 
-[id="rosa-new-features_{context}"]
-== New features and enhancements
+. Download the file from the link:https://console.redhat.com/openshift/downloads[Hybrid Cloud Console].
```

Comment:
This feels like it should be a procedure module and the commands and outputs should be in code blocks. Additionally, I'm not sure the release notes (or "what's new") are a place for a procedure. Would this fit better elsewhere in the docs, and add an xref or Additional resources to link to it?